public_html/index.html

```diff
@@ -8,6 +8,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>Mozilla – The Future of Web Games</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="games.mozilla.org">
+    <meta name="twitter:domain" content="mozilla.org">
```

Comment:
it looks like this tag is deprecated now

Reply:
removed!